src/handlers/serenity.rs

```diff
@@ -125,27 +125,46 @@ impl SerenityHandler {
                 .create_application_command(|command| {
                     command
                         .name("play")
-                        .description("Adds a track to the queue")
+                        .description("Add a track to the queue")
                         .default_permission(true)
                         .create_option(|option| {
                             option
-                                .name("query")
-                                .description("The media to play")
-                                .kind(ApplicationCommandOptionType::String)
-                                .required(true)
+                                .name("default")
+                                .description("Place this song at the bottom of the queue")
```

Comment:
Sorry, just a final change, I missed this. 😓

Please rename default to **end** and change description to **"Add a track to the end of the queue"** - it'll make it more in line with the rest of the descriptions.